requirements/lint.txt

```diff
@@ -1,8 +1,13 @@
 -r pip.txt
 maxcdn
 astroid
-pylint
-# prospector==0.12.6 currently has issues with pydocstyle
-prospector==0.12.5
+
+# pylint 1.8.0 is having problems:
+# File "/home/humitos/.pyenv/versions/2.7.14/envs/pylint/lib/python2.7/site-packages/pylint_django/plugin.py", line 22, in register
+#     start = name_checker.config.const_rgx.pattern[:-2]
+# AttributeError: 'NoneType' object has no attribute 'pattern
+pylint==1.7.5
+
+prospector
```

Comment:
Shouldn't prospector also be pinned?

Reply:
"Maybe".

Read my comment below in this PR.

Reply:
I'm torn on whether we pin these or not. We are fighting broken testing dependencies at least twice a month now. Pinning does help avoid this, but I don't think there is a strong reason to pin otherwise.